Wiki/starting/starting-leaving.md

```diff
@@ -0,0 +1,146 @@
+As model output includes raw output, restart files and post-processed output, they should all be treated differently. usually if keeping raw output this is stored on tape, it is a bit harder to retrieve but tape is less expensive than other storage. Only some of the restart files are kept (if any). Ideally, post-processed should be published if it has been used for publication or if it's going to be shared with others. 
+
+**Output of lengthy processing**
+
+ It might feel necessary to keep those files but we would argue it is usually not worth the cost of the storage unless there is a clear indication they will be used again soon. The first reason is that with current modern programming techniques a lot of lengthy processing can be shortened significantly. The second reason is the very real cost of the storage is not worth the very hypothetical time saved in the future.
+
+
+## Publishing data and code
+
+You should publish your files if the files are frequently used by several other persons. Ideally, that data should be published as soon as its usefulness to others is clear, not just when you leave.
+
+If your files are at NCI, that is all you need to do. If your files are held at your institution, you may need to make sure there is a copy that is accessible by others and owned by someone who is likely to stay for years to come. This might depend on your institution data services.
+
+**Sharing files without publication**
+
+Some files might be useful to a small cluster of people but not be worth publishing. In that case, you simply need to change the ownership of the files: get the files to be owned by someone who stays after you.
+
+If the files are at NCI, you can not change the files' ownership. You and the Lead CI of the project owning the files will need to contact help@nci.org.au so NCI staff can do it for you.
+
+```{warning}
+Often researchers shared data with others without publishing it even when this data is used for new publication, this should be avoided at any cost. It does create an issue for the creator of the data who won't get recognition and for the user who might struggle to refer to the exact data when publishing a paper. As it is now a requirement to publish data associated with a paper it is always bets to publish before sharing!
+```
+
+## Your NCI account
+
+When you leave, you will keep your NCI credentials IF you keep your contact details up to date through https://my.nci.org.au
+
+Make sure to quit all NCI projects you don't need access to anymore. You can be ruthless as it is quite easy to regain access to those projects later on if needed.
+
+Some projects have strict license terms attached (e.g. access). Your membership might be revoked at any time after you leave the Centre if you haven't negotiated extended access to those projects. Please contact the Lead CI or the project manager to discuss your needs.
```

Comment:
```suggestion
Some projects have strict license terms attached (e.g., ACCESS). Your membership might be revoked any time after you leave the Centre if you haven't negotiated extended access to those projects. Please contact the Lead CI or project manager to discuss your needs.
```

Reply:
the access project is actually all lower case, I'll commit this and then change it later